lib/mockbuilder.py

```diff
@@ -195,6 +195,14 @@ def copy_results(self, package):
             self.timestamp, package.name)
         self._copy_rpms(package.build_cache_dir, package_build_results_dir)
 
+    def create_repository(self):
+        """
+        Create yum repository in build results directory.
+        """
+        build_results_dir = os.path.join(
+            CONF.get('common').get('result_dir'), 'packages', self.timestamp)
+        utils.run_command("createrepo %s" % build_results_dir)
```

Comment:
IMHO it makes more sense to put this in utils.py or a new yum_repository.py. You could put the yum repository config file creation there too in the later patch which does it. This module should contain *only* Mock-specific features. But up to you.